lib/template.js

```diff
@@ -104,9 +109,25 @@ template.questions = function(templateBody, overrides) {
       if ('MaxValue' in parameter) valid = valid && Number(input) <= parameter.MaxValue;
       if (parameter.AllowedPattern) valid = valid && (new RegExp(parameter.AllowedPattern)).test(input);
       if (parameter.Type === 'List<Number>') valid = valid && input.split(',').every(isNumeric);
+
       return valid;
     };
 
+    question.filter = function(input) {
+      var done = this.async();
+
+      if (overrides.kms && parameter.Description.indexOf('[secure]') > -1 && input.slice(0, 7) !== 'secure:') {
+        return kms.encrypt({
+          KeyId: overrides.kms,
+          Plaintext: input
+        }, function(err, encrypted) {
+          done(err, 'secure:' + encrypted.CiphertextBlob.toString('base64'));
```

Comment:
If there is an `err`, usually the aws-sdk does not return a second argument. In that case `encrypted. CiphertextBlob` would throw an exception.